From 1eb9b6b37b496048e79f6802ab199bc94bb1d7ca Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 27 Jan 2016 14:17:23 +0000
Subject: [PATCH 0728/1077] MANTIS 0037186 codes.idl V1.3.3

CUSS-Spec: missing definition of DS_TYPES_KEY

changed value of DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP



git-svn-id: svn://localhost/SelfServiceCommon/trunk@971 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 06c4df0b..8597a726 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -32,6 +32,11 @@
 //  @li added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
 //  @li the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
 //                 
+//  @date 27.01.2016
+//  @li V1.3.3
+//  @li Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
+//  @li changed values for DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
+//                 
 //----------------------------------------------------------------------------
 
 #ifndef CODES_IDL
@@ -145,10 +150,9 @@ module datastatus
     const long DS_TYPES_EPAYMENT                 = 23000; /**< E-Payment: CuPaySchema.xsd */
     const long DS_TYPES_ILLUMINATION             = 24000; /**< LED CUSS_ILLUMINATION.xsd */
 
-    const long DS_TYPES_KEY                      = 25100; /**< key pressed */
-    const long DS_TYPES_KEY_DOWN                 = 25110; /**< key initially pressed */
-    const long DS_TYPES_KEY_UP                   = 25120; /**< key released */
-
+    const long DS_TYPES_KEY                      = 26000; /**< key pressed */
+    const long DS_TYPES_KEY_DOWN                 = 26010; /**< key initially pressed */
+    const long DS_TYPES_KEY_UP                   = 26020; /**< key released */
 };
 
 /**
-- 
2.41.0.windows.1

